src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -97,6 +98,15 @@ public function boot()
         }
     }
 
+    public function shutdown()
+    {
+        parent::shutdown();
+
+        if (class_exists(AbstractStaticOption::class)) {
+            AbstractStaticOption::reset();
```

Comment:
would adding this call to `DoctrineType::reset()` fix the leak for you?

Reply:
No, that didn't work.